<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPrefCaja</class>
 <widget class="QWidget" name="FormPrefCaja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="BGCierre">
     <property name="title">
      <string>Cierre de caja</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="RBDiferencia">
        <property name="text">
         <string>No permitir cierre de caja si hay diferencia con el conteo diario</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RBAjuste">
        <property name="text">
         <string>Ajustar saldo de caja segun conteo siempre</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="CkBNoGastos">
     <property name="text">
      <string>No permitir gastos si no hay fondos suficientes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="CkBResumen">
     <property name="text">
      <string>Siempre imprimir resumen de cierre de caja</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GBResumen">
     <property name="title">
      <string>Resumen</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="CkBFirma">
        <property name="text">
         <string>Agregar firma y autorizante al final del resumen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CkBLogo">
        <property name="text">
         <string>Agregar logo en la cabecera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CkBResponsable">
        <property name="text">
         <string>Mostrar columna de responsable</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GBCaja">
     <property name="title">
      <string>Caja a usar en compras, ventas y gastos</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Seleccióne la caja que se utilizará para ingresar las ventas al contado y las compras al contado. También será la que aparecerá como predeterminada en el agregado de un gasto.&lt;/span&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Caja:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="CBCaja">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
